dotcom-rendering/src/web/components/ContainerTitle.tsx

```diff
@@ -91,11 +91,13 @@ export const ContainerTitle = ({
 
 	const now = new Date();
 	const locale = editionId && getEditionFromId(editionId).locale;
-
 	return (
 		<div css={marginStyles}>
 			{url ? (
-				<a css={[linkStyles, bottomMargin]} href={url}>
+				<a
+					css={[linkStyles, bottomMargin]}
+					href={`https://www.theguardian.com/${url}`}
+				>
```

Comment:
I noticed that this component `ContainerTitle` is used in both FrontSection.tsx & Section.tsx, could we have conflicting behaviours here?

Might be worth checking where else this URL property is used (if anywhere) in articles etc too, to see if this change should be made across the board or just for Fronts!